lib/src/marker_cluster_layer.dart

```diff
@@ -28,18 +28,18 @@ class _MarkerClusterLayerState extends State<MarkerClusterLayer>
     with TickerProviderStateMixin {
   Map<int, DistanceGrid<MarkerClusterNode>> _gridClusters = {};
   Map<int, DistanceGrid<MarkerNode>> _gridUnclustered = {};
-  MarkerClusterNode _topClusterLevel;
-  int _maxZoom;
-  int _minZoom;
-  int _currentZoom;
-  int _previousZoom;
-  double _previousZoomDouble;
-  AnimationController _zoomController;
-  AnimationController _fitBoundController;
-  AnimationController _centerMarkerController;
-  AnimationController _spiderfyController;
-  MarkerClusterNode _spiderfyCluster;
-  PolygonLayer _polygon;
+  late MarkerClusterNode _topClusterLevel;
+  int? _maxZoom;
+  late int _minZoom;
+  int? _currentZoom;
+  int? _previousZoom;
+  double? _previousZoomDouble;
```

Comment:
I think we should consider setting _previousZoom and _previousZoomDouble in initState so that they can be non-nullable.

Reply:
done